packages/vsce/README.md

```diff
@@ -124,71 +124,71 @@ Right-click a CICS profile and select `Manage Profile` to show profile options.
 
 <p align="center">
 <img src="/packages/vsce/docs/images/region-filter.gif" alt="Zowe CICS Explorer Filter" width="700px"/>
 </p>
 
-**Tip:** To apply multiple filters, separate entries with a comma. You can append any filter with an \*, which indicates wildcard searching.
+**Tip:** To apply multiple filters, separate entries with a comma. You can append any filter with an `\*`, which indicates wildcard searching.
 
-### Show and Filter Resources in a Plex
+### Show and filter resources in a plex
 
-In the same way as filtering resources in a region, you can apply a filter on resources in a plex. Use the filter and clear filter buttons against the Regions tree element to manage filters across the whole set of regions and resources.
+In the same way as filtering resources in a region, you can apply a filter on resources in a plex. Use the **Filter** and **Clear Filter** buttons against the Regions tree element to manage filters across the whole set of regions and resources. **[Should "filter" and "clear filter" be "search" and "clear search"? the icons looks like "search" icons rather than "filter" icons. From what I see, seems like we're using "search" and "filter" interchangeably and we should use only one of these terms.]**
 
 <p align="center">
 <img src="/packages/vsce/docs/images/plex-filter.gif" alt="Zowe CICS Explorer Filter" width="700px"/>
 </p>
 
-### Show and Filter Resources in an 'All' Resource Tree
+### Show and filter resources in an 'All' resource tree
 
-CICSplexes contain an `All Programs`, `All Local Transactions`, `All Local Files` and `All Tasks` trees which show resources from all regions in the plex.
+CICSplexes contain an `All Programs`, `All Local Transactions`, `All Local Files`, and `All Tasks` trees which show resources from all regions in the plex.
 
-To view resources under these trees, use the search icon <img src="/packages/vsce/docs/images/resource-filter.png" width="16px"/> inline with the tree and apply a filter.
+To view resources under these trees, use the **Search** icon <img src="/packages/vsce/docs/images/resource-filter.png" width="16px"/> inline with the tree and apply a filter.
 
 <p align="center">
 <img src="/packages/vsce/docs/images/all-resources.gif" alt="Zowe CICS Explorer Filter" width="700px"/>
 </p>
 
-If the applied filter results in over 500 records, either change the filter to narrow down the search, or click the `view X more ...` item to retrieve 'X' more resources.
+If the applied filter results in more than 500 records, either change the filter to narrow down the search, or click the `view X more ...` item to retrieve 'X' more resources.
 
-**Tip:** The default 500 count can be modified via the `Record Count Increment` property in Settings (UI).
+**Tip:** The default 500 count can be modified via the `Record Count Increment` property in Settings (UI). **[What does "(UI)" mean here? I don't see it in the screenshot.]**
```

Comment:
@davenice - please respond

Reply:
UI means the settings page in VS Code rather than the config file, I imagine. Please check my clarified wording @anaxceron

Reply:
(In fact I found another place where we talk about the settings and have now taken the wording from there!)